### DISCUSSION TITLE
Is there a time limit on calendar events?

### DISCUSSION THREAD

**Discussion by danceswithcats** [Etar-Group/Etar-Calendar#1048] [Q&A] [answered]
Hi,

Thank you all for this app. I use it on a Lineage OS One Plus that would otherwise have become ewaste two years ago.

I want to ask whether events are cleared after a given amount of time. I seem to stop seeing events after six months. I've checked the Calendar settings and Davx5 settings and can't find anything about this. Events on my Next cloud calendar go back to when I set up my server, several years ago.

Sometimes I need to check back to a past event. Yesterday, I wanted to know how long it was since my last covid vaccination, for instance. I had to log in to my online calendar via the browser for that.

Anyway, thanks again.

Kind regards,

Peter

**Comment by jspricke** [comment, marked as answer]
Davx5 has a per DAV account settings to ignore older events, default to 90 days.

**Comment by danceswithcats** [comment]
Oops! Sorry I missed that. Thank you.

3 Dec 2021 07:33:09 Jochen Sprickerhof ***@***.***>:
